src/index.html

```diff
@@ -7,7 +7,287 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <img src="images/phone.png"
+           alt=""
+           class="header__phone-img">
```

Comment:
```suggestion
      <img
        src="images/phone.png"
        alt=""
        class="header__phone-img"
      >
```

Твой вариант в принципе тоже ничего выглядит, но давай придерживаться единого стандарта)

Reply:
p.s. думаю стоит обернуть эту картинку в ссылку, и сделать чтобы при клике выскакивало предложение позвонить по номеру